api/services/ocean/spark/parameters/query/orderBy.yaml

```diff
@@ -0,0 +1,9 @@
+in: query
+name: orderBy
+required: false
+description: The field on which to sort
```

Comment:
We only support sorting by CREATED_AT, it seems? 
So we could say:

> The field on which to sort. For now, we only support filtering on CREATED_AT.

Reply:
Done!